README.md

```diff
@@ -474,6 +474,8 @@ There are only rendered in the target page when the version filter matches the c
 
 Partial Convention: `filePrefix_key_nugetAlias_version.partial.md`
 
+Usually the NuGet alias will be the same as the component name. When a component has multiple NuGet aliases, a proper alias should be used, not the component name. 
```

Comment:
Yes. I think @weralabaj that I can't up with a better description that makes this differentiation explicit :(